build.gradle

```diff
@@ -23,6 +23,50 @@ buildscript {
 
 plugins {
     id("io.gitlab.arturbosch.detekt").version("1.0.0-RC16")
+    id("com.nimbledroid.profiler").version("1.2.1")
+}
+
+// Placeholder for ND plugin configuration.
+//
+// The ND plugin expects this to be where gradle
+// stores configuration information. Variables
+// are set by the setNimbledroidConfiguration task.
+nimbledroid {
```

Comment:
I think this config should move to the `app/build.gradle` file because I don't think it makes sense to use nimbledroid on anything but the final app APKs.

If this config stays in this file, it should probably happen below `allProjects`, I think (that structure makes more sense to me).

Reply:
Good call! I will definitely move it!